arkouda/pdarrayIO.py

```diff
@@ -122,6 +122,7 @@ def read(filenames : Union[str, List[str]],
     and read all of them. Use ``get_datasets`` to show the names of datasets
     to HDF5 files.
     """
+    print(filenames)
```

Comment:
I'm not sure we want to print out the filenames. Sometimes there can be thousands of them.

Reply:
Agreed, this was left over from some debugging I was doing. I just pushed up the removal of the print.